test/system/clients_test.rb

```diff
@@ -63,11 +66,15 @@ class ClientsTest < ApplicationSystemTestCase
     fill_in 'Street', with: 'Sihlstrasse 131'
     fill_in 'Zip', with: '8002'
     fill_in 'City', with: 'Zürich'
+    within '#languages' do
+      page.choose('client_language_skills_attributes_0_level_good')
```

Comment:
Interactive browser tests should always use user-visible labels rather than HTML ID's and classes. Of course this isn't always feasible, but maybe you can get it working with `select "Gut", from: "Niveau"` or something like that? You'll probably still need the `within '#languages'` though ;-)